code/modules/client/preference/preferences.dm

```diff
@@ -2133,6 +2137,13 @@ GLOBAL_LIST_INIT(special_role_times, list( //minimum age (in days) for accounts
 					INVOKE_ASYNC(user.client, /client.proc/edit_2fa)
 					return // We return here to avoid focus being lost
 
+				if("screentipmode")
+					user.client.toggle_onscreen_tips()
+
+				if("screentipcolor")
```

Comment:
Would prefer these have some underscores. You'll have to modify the hypertext links on line 493 and 494 as well.
```suggestion
				if("screentip_mode")
					user.client.toggle_onscreen_tips()

				if("screentip_color")
```